src/Tests/Outbox/LogicalOutboxBehaviorTests.cs

```diff
@@ -32,11 +33,17 @@ public async Task SetUp()
             var account = CloudStorageAccount.Parse(ConnectionStringHelper.GetEnvConfiguredConnectionStringForPersistence(tableApiType));
 
             client = account.CreateCloudTableClient();
-            tableName = nameof(LogicalOutboxBehaviorTests).ToLower();
+            tableName = $"{Path.GetFileNameWithoutExtension(Path.GetTempFileName())}{DateTime.UtcNow.Ticks}{nameof(LogicalOutboxBehavior)}".ToLowerInvariant();
             cloudTable = client.GetTableReference(tableName);
             await cloudTable.CreateIfNotExistsAsync();
         }
 
+        [TearDown]
+        public async Task Teardown()
+        {
+            await cloudTable.DeleteIfExistsAsync();
```

Comment:
Should the task just be returned to be consistent with the approach other cleanups are taking?